api/api.go

```diff
@@ -55,48 +74,198 @@ type ConfigResponse struct {
 	SourceYaml string
 	OutputYaml string
 
-	GQLResponseErrors
+	Errors GQLErrorsCollection
```

Comment:
This `Errors` field is in the wrong spot. It needs to be at the top level of the `BuildConfigResponse` struct, as a sibling to `Data` field.